src/test/java/com/agile/vision/summer/practice/services/MonitorServiceTest.java

```diff
@@ -0,0 +1,81 @@
+package com.agile.vision.summer.practice.services;
+
+import com.agile.vision.summer.practice.entities.Monitor;
+import com.agile.vision.summer.practice.repositories.MonitorRepository;
+import org.junit.Test;
+import org.junit.jupiter.api.extension.ExtendWith;
+import org.junit.runner.RunWith;
+import org.mockito.InjectMocks;
+import org.mockito.Mock;
+import org.mockito.Mockito;
+import org.mockito.runners.MockitoJUnitRunner;
+import org.mockito.junit.jupiter.MockitoExtension;
+import org.springframework.boot.test.context.SpringBootTest;
+import org.springframework.test.context.ActiveProfiles;
+
+import java.util.*;
+import java.util.logging.Logger;
+
+import static org.junit.Assert.assertEquals;
+import static org.mockito.Mockito.times;
+import static org.mockito.Mockito.verify;
+
+@ActiveProfiles("test")
+@ExtendWith(MockitoExtension.class)
```

Comment:
Please use Junit4 or Junit5 in the your project and one test file. Now You use mixed. You use ExtendWith from Junit5, but use ,method format from junit4

Reply:
import org.junit.Test; - it is from junit4